src/main/java/com/github/firmwehr/gentle/GentleCompiler.java

```diff
@@ -24,38 +21,7 @@
 
 public class GentleCompiler {
 
-	public static final boolean IS_SPEEDCENTER;
-
-	// DO NOT MOVE BELOW LOGGER DECLARATION
-	static {
-		// check for speedcenter run and change logging format
-		if (System.getProperty("speedcenter.true") != null) {
-			try (var in = GentleCompiler.class.getResourceAsStream("/log4j2_speedcenter.xml")) {
-				if (in == null) {
-					throw new IOException("Could not find speedcenter logger configuration file");
-				}
-
-				var configSource = new ConfigurationSource(in);
-				Configurator.initialize(null, configSource);
-				//noinspection UseOfSystemOutOrSystemErr
-				System.err.println("Detected speedcenter! Switched to speedcenter logger config");
```

Comment:
I would prefer to have some kind of feedback, indicating the current state of the logger. Makes it easier to debug CI runs with unexpected logger behavior.